test/clients/clientUpdate.spec.js

```diff
@@ -0,0 +1,109 @@
+import { expect } from 'chai';
+const chance = require('chance').Chance();
+import * as clientHelper from '../../helpers/clientHelper';
+
+describe('UPDATE CLIENT', () => {
+  let resClientCreate,
+    idClient,
+    updatedClient,
+    getClient,
+    getUpdClient;
+
+  let name = chance.name();
```

Comment:
Hey guys, I had to use this pattern because ClientCreate Function expects 4 arguments. I would like to suggest to pass arguments in a single object. I can do that please let me know and I can create a new PR. @dumpdusty @innazhig @ZdanovichOleg  @RafailFerre